src/components/record/related-table-actions.tsx

```diff
@@ -519,30 +510,6 @@ const RelatedTableActions = ({
       setShowPureBinarySpinner(true);
 
       validateSessionBeforeMutation(() => {
-        const deleteResponse = (response: any) => {
-          setShowPureBinarySpinner(false);
-
-          // Show modal popup summarizing total # of deletions succeeded and failed
-          response.clickOkToDismiss = true;
-
-          // TODO: - improve partial success and use TRS to check delete rights before giving a checkbox
-          //       - some errors could have been because of row level security
-          dispatchError({
```

Comment:
If you pause the code at this line, you will see the `response` parameter in `deleteResponse` will be the "Batch Unlink Summary" which we want to avoid showing.